### DISCUSSION TITLE
Feature Request: Application Ignore List

### DISCUSSION THREAD

**Discussion by josh-stephens** [pieces-app/support#410] [Ideas]
An application ignore list would make me a lot more confident about the context that Pieces is gathering. I want to be able to explicitly tell Pieces not to look at certain windows such as my Bitwarden password manager window that contains sensitive entries even if the passwords aren't showing.

Because: what if I'm in Blended mode and select some context for a given time period, and there's some particular piece of information that the behind-the-scenes ML and algorithms have overlooked that could then make its way into GPT or Claude?

**Comment by josh-stephens** [comment]
One stopgap measure would be the ability to set a global hotkey for pausing Live Context.

**Comment by rosie-at-pieces** [comment]
Great feedback @josh-stephens - this is something we're actively working on! i've added your specific feedback to the issue.

https://github.com/pieces-app/os_server/issues/807
